src/app-profile.ts

```diff
@@ -29,7 +29,7 @@ export interface AppProfileOptions {
    * value is required when creating the app profile and optional when setting
    * the metadata.
    */
-  routing?: 'any' | Cluster;
+  routing?: 'any' | Cluster | Set<Cluster> | Set<string>;
```

Comment:
let's add a test for this new option `Set<string>`

Reply:
You can find this under the test that says `should create a profile with multiple clusters using strings`.